### DISCUSSION TITLE
zepetopackage 파일을 유니티로 여는법

### DISCUSSION THREAD

**Discussion by JeongYire** [naverz/zepeto-studio-kor#317] [5. Q&A]
zepetopackage 파일을 유니티로 여는법 은 따로 없을까요?
유니티 작업하다가 실수로 날라가버려서... 그당시 zepetopacakage  파일은 존재하는데 유니티로 여는방법이 없을까요?

**Comment by x4cf** [comment]
불가능할것같습니다

**Comment by hyeonjin-jo** [comment]
안녕하세요.

안타깝게도 패키지파일을 유니티에서 오픈하는 기능은 제공하고 있지 않습니다.
